src/url.js

```diff
@@ -461,6 +460,24 @@ function removeAmpJsParamsFromSearch(urlSearch) {
   return search ? '?' + search : '';
 }
 
+/**
+ * Removes parameters with param name and returns the new search string.
+ * @param {string} urlSearch
+ * @param {string} paramName
+ * @return {string}
+ */
+export function removeParamsFromSearch(urlSearch, paramName) {
```

Comment:
(can be done in a separate PR) worth to expand the method to take more paramName , so that we can migrate the above `removeAmpJsParamsFromSearch` to this.

Reply:
Right I put a TODO here. Updated the TODO.